### DISCUSSION TITLE
Search Results limited to 20 results

### DISCUSSION THREAD

**Discussion by Joffcom** [npm/feedback#769] [Website] [answered]
When searching on npmjs.com it is only returning 20 results, Doesn't seem to matter what search term is used or browser.

**Comment by filfreire** [comment]
@Joffcom seems like a recent bug.

If you try to add a `page` query parameter and change its value, e.g. `&page=5`, each page seems to return a different set of 20 results, BUT there's no indication on the UI that what we see is split into pages.

**Comment by MylesBorins** [comment, marked as answer]
hey @Joffcom thanks for surfacing. I've surfaced to our engineering team who are going to dig in and figure out what's going on so we can fix this.

**Reply by MylesBorins** [reply]
Just confirming this is now fixed. TYVM for reporting!
